src/main/java/apoc/export/cypher/formatter/AbstractCypherFormatter.java

```diff
@@ -102,20 +102,19 @@ public void buildStatementForNodes(String nodeClause, String setClause,
 				.collect(Collectors.groupingByConcurrent(keyMapper));
```

Comment:
is there any way we could stream this without collecting everything into a huge list?
This could be many millions of nodes?